infrastructure/api/handler/login_handelr.go

```diff
@@ -0,0 +1,43 @@
+package handler
+
+import (
+	"log"
+
+	"tat_gogogo/infrastructure/middleware"
+	"tat_gogogo/interface/controller"
+
+	"github.com/gin-gonic/gin"
+)
+
+/*
+LoginHandler is a function for gin to handle login api
+*/
+func LoginHandler(c *gin.Context) {
+	authMiddleware, err := middleware.NewAuthMiddleware()
+	if err != nil {
+		c.Status(500)
+		log.Fatal("JWT Error:" + err.Error())
+
+	}
+
+	studentID := c.PostForm("studentID")
+	password := c.PostForm("password")
+
+	loginController := controller.NewLoginController(studentID, password)
+	result, err := loginController.Login()
+
+	if err != nil {
+		log.Panicln("failed to fetch login cookie")
```

Comment:
`log.Panicln` will print your message then call the `panic` function, which, in summary, will also crash your program. I don't see the need to crash the program in the case of any malfunctions occurring in the login function

Reply:
same as here, do I just use the regular print? Is there any rule that I have to fellow.

Reply:
Yep. Pretty much the same as the previous comment.
I think for the meantime this is okay, but we will have to tackle it in the Issue concerning Error Handling.